xarray/backends/common.py

```diff
@@ -161,8 +161,12 @@ def add(self, source, target):
 
     def sync(self):
         if self.sources:
-            import dask.array as da
-            da.store(self.sources, self.targets)
+            import dask, dask.array as da
+            import threading
```

Comment:
We can move `import threading` to the top of the file -- we only have to import dask here because it isn't a required dependency